src/Uno.UI/UI/Xaml/Controls/Image/NativeImage.iOSmacOS.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Analyzer 'SonarAnalyzer.Rules.CSharp.ConditionEvaluatesToConstant' threw an exception of type 'SonarAnalyzer.SymbolicExecution.SymbolicExecutionException' with message 'Error processing method: ImageSize ## Method file: NativeImage.iOSmacOS.cs ## Method line: 19,26 ## Inner exception: System.NullReferenceException: Object reference not set to an instance of an object ##   at SonarAnalyzer.SymbolicExecution.CSharpExplodedGraph.VisitDefaultExpression (Microsoft.CodeAnalysis.SyntaxNode instruction, SonarAnalyz](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5554394)